packages/perseus/src/widgets/grapher/grapher.cypress.ts

```diff
@@ -55,8 +56,12 @@ describe("Grapher widget", () => {
 
             // Assert
             cy.then(() => {
-                const state = getRenderer().guessAndScore();
-                expect(state[1]).toStrictEqual({
+                const userInput = getRenderer().getUserInputMap();
+                const score = scorePerseusItemTesting(
+                    absoluteValueQuestion,
+                    userInput,
+                );
+                expect(score).toStrictEqual({
```

Comment:
That's correct, because `toHaveBeenAnsweredCorrectly` is a Jest custom matcher. Although our Cypress tests _look_ kinda like Jest tests, they aren't, and they just happen to have similar assertion APIs. We didn't replicate the custom matchers for the Renderer in Cypress.